CHANGELOG.md

```diff
@@ -35,6 +35,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fixed a bug where java refiner would emit enums without normalizing the name.
 - Fixed bug in request configuration class constructor PHPDoc.
 - Fixed a bug where ruby refiner would emit enums without normalizing the name.
+- Fixed a bug where java refiner would emit method's parameters types without normalizing the name.
```

Comment:
needs to be move to the unreleased section.

Reply:
gentle reminder on this one @andreaTP

Reply:
ouch, sorry 😅 , on it

Reply:
should be good now